pkgs/common-updater/unstable-updater.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, writeScript, coreutils, git, common-updater-scripts }:
+
+# This is an updater for unstable packages that should always use the latest
+# commit.
+{ attrPath
+, url # The git url, usually just src.url
+}:
+
+let
+  updateScript = writeScript "unstable-update-script.sh" ''
+    #! ${stdenv.shell}
```

Comment:
With `writeShellScript`, this line can be omitted.

Reply:
Done